hack/jenkins/e2e.sh

```diff
@@ -106,6 +106,22 @@ Addon\
 |multiport\sendpoints\
 "
 
+# release-0.21 GCE skip list
+RELEASE_0_21_GCE_SKIP_TEST_REGEX="\
```

Comment:
Ah, right, we actually checkout e2e.sh at the version we're running.

So yes, jussec.